src/toil_scripts/adam_gatk_pipeline/align_and_call.py

```diff
@@ -322,67 +176,58 @@ def static_dag(job,
     work_dir = job.fileStore.getLocalTempDir()
 
     # what region is our bucket in?
-    if bucket_region == "us-east-1":
-        bucket_region = ""
+    if inputs.bucket_region == "us-east-1":
```

Comment:
No, you're right. We're using s3:// urls so that input is never used. I'll get rid of it.